### PR TITLE
Define simple and complex types

### PR DESCRIPTION
**Pull request by Rick-Anderson**
[Internal review URL](https://review.learn.microsoft.com/en-us/aspnet/core/mvc/advanced/custom-model-binding?view=aspnetcore-7.0&branch=pr-en-us-27415#model-binding-simple-and-complex-types)

Replaces PR #26975

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@brunolins16

File: aspnetcore/mvc/advanced/custom-model-binding.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ Model binding allows controller actions to work directly with model types (passe
...
 
-## Model binding review
+## Model binding simple and complex types
 
-Model binding uses specific definitions for the types it operates on. A *simple type* is converted from a single string in the input. A *complex type* is converted from multiple input values. The framework determines the difference based on the existence of a `TypeConverter`. We recommend you create a type converter if you have a simple `string` -> `SomeType` mapping that doesn't require external resources.
+Model binding uses specific definitions for the types it operates on. A *simple type* is converted from a single string from a <xref:System.ComponentModel.TypeConverter> or a `TryParse` method. A *complex type* is converted from multiple input values. The framework determines the difference based on the existence of a `TypeConverter` or `TryParse`. We recommend creating a type converter or using `TryParse` for a `string` to `SomeType` conversion that doesn't require external resources or multiple inputs.
```
Comment (brunolins16) [inline aspnetcore/mvc/advanced/custom-model-binding.md:25]: Might be me but this `from a single string from a <xref:System.ComponentModel.TypeConverter> or a `TryParse` method` sounds strange.

Comment (Rick-Anderson) [inline aspnetcore/mvc/advanced/custom-model-binding.md:25]:
```suggestion
Model binding uses specific definitions for the types it operates on. A *simple type* is converted from a single string using <xref:System.ComponentModel.TypeConverter> or a `TryParse` method. A *complex type* is converted from multiple input values. The framework determines the difference based on the existence of a `TypeConverter` or `TryParse`. We recommend creating a type converter or using `TryParse` for a `string` to `SomeType` conversion that doesn't require external resources or multiple inputs.
```

Comment (Rick-Anderson) [reply aspnetcore/mvc/advanced/custom-model-binding.md:25]: @brunolins16 you're right, how's this?

Comment (brunolins16) [reply aspnetcore/mvc/advanced/custom-model-binding.md:25]: Much better.

**Review by brunolins16 — APPROVED** [review]
Looks good after you apply your suggested change

**Pull request merged**